apps/extension/src/ui/domains/Asset/Buy/BuyTokensForm.tsx

```diff
@@ -140,6 +140,27 @@ export const BuyTokensForm = () => {
     address
   )
 
+  const autoSelectFirstAccountInit = useRef(false)
```

Comment:
with metadata/opt-in PR this useAccounts will now suspense until all accounts are loaded, using a ref should become unnecessary then.
might want to add a TODO comment to remove it in the future